### PR TITLE
PTM: Changed the ptm services  to be like the IR, HID, and APT services.

### PR DESCRIPTION
**Pull request by purpasmart96**
I hope Subv doesn't mind... Because I really like how for the HID services have all their stuff in one area, so you don't have to duplicate the same functions all over the place in the related services to it.

### CODE REVIEW

**Comment by Subv** [comment]
It's OK with me, but you'll have to change the doxygen comments in some of the functions you changed.

File: src/core/hle/service/ptm/ptm.h [hunk 1]
```diff
@@ -41,19 +42,38 @@ struct GameCoin {
  * that controls another part of the HW.
  * @returns 1 if the battery is charging, and 0 otherwise.
```
Comment (Subv) [inline src/core/hle/service/ptm/ptm.h]: Here, for example, this function doesn't return anything anymore.

Comment (purpasmart96) [inline src/core/hle/service/ptm/ptm.h]: Will do.

**Comment by purpasmart96** [comment]
Done

**Comment by bunnei** [comment]
LGTM

**Pull request merged**